### ISSUE TITLE
Assign a pattern to a stream instead of calling p(..., ...)

### ISSUE THREAD

**Issue by karstenj** [tidalcycles/vortex#41]
It would be better to assign a pattern to a stream similar to tidal. 

Currently in vortex: p("d1", s("bd"))
Tidal: d1 $ s("bd")
Proposed solution: d[1] = s("bd")

The '1' in the square brackets could be any hashable object in Python (eg. d['mystream'], d[(1, 2)], ...).

add in boot.py:
```
class __Streams:
    def __setitem__(self, key, value):
        p(key, value)

d = __Streams()
```

**Comment by yaxu** [comment]
Yes definitely an improvement, we might find a alternative UI that's nicer still later, though.

**Comment by karstenj** [comment]
An alternative UI could be a Jupyter notebook/lab. It already provides a lot of functions out of the box:
- Evaluate patterns in cells
- Syntax highlighting
- Sharing of notebooks for collaborative coding (https://jupyterlab.readthedocs.io/en/stable/user/rtc.html)
- Visualization capabilities

This is an example of a jupyter notebook running in VsCode:
https://drive.google.com/file/d/1s6tGc42-oKilGVSpZEA2sqHrsgjx8p08/view?usp=share_link

**Comment by karstenj** [comment]
Has been merged by #43
